src/Core/Validation/ValidationResult.php

```diff
@@ -50,204 +52,185 @@ class ValidationResult
     /**
      * Is the result valid or not.
      * Note that there can be non-error messages in the list.
-     *
-     * @var bool
      */
-    protected $isValid = true;
+    protected bool $isValid = true;
 
     /**
      * List of messages
-     *
-     * @var array
      */
-    protected $messages = [];
+    protected array $messages = [];
+
+    /**
+     * The class of the DataObject being validated.
```

Comment:
```suggestion
     * The class of the model being validated.
```

Reply:
Updated, have also changed `dataClass` to `modelClass`